front/src/applications/stdcmV2/components/StdcmConfig.tsx

```diff
@@ -77,8 +80,19 @@ const StdcmConfig = ({
   };
 
   const startSimulation = () => {
-    if (pathfinding?.status === 'success' && !formErrors) {
+    const isPathfindingFailed = !!pathfinding && pathfinding.status !== 'success';
+    const formErrorsStatus = checkStdcmConfigErrors(
+      isPathfindingFailed,
+      origin,
+      destination,
+      compact(pathSteps),
+      t
+    );
+    if (pathfinding?.status === 'success' && !formErrorsStatus) {
       launchStdcmRequest();
+    } else {
+      console.error('The form is not valid:', { pathfinding, formErrorsStatus });
```

Comment:
We should keep it a warning ?

```suggestion
      console.warn('The form is not valid:', { pathfinding, formErrorsStatus });
```

Reply:
Yes, I want to keep it for the user tests. I can add a comment :+1: